Source/Core/DolphinQt/Config/MouseSettingsWidget.cpp

```diff
@@ -0,0 +1,142 @@
+  m_sensitivity_control->setDecimals(2);
+  m_sensitivity_control->setSingleStep(0.5);
+  m_sensitivity_control->setWrapping(true);
+  QString sensitivity_tool_tip{
+      tr("Adjusts how quickly the mouse cursor moves the Cursor X/Y +/- inputs."
+         "\nChanges the distance at which your mouse snaps to notches. May make your snapping feel "
+         "weird."
+         "\n\n1.0(default) sensitivity maps the mouse to the full screen"
+         "\n\nRecommended value: 7.5")};
+  m_sensitivity_control->setToolTip(sensitivity_tool_tip);
+
+  QLabel* sensitivity_label = new QLabel{};
+  sensitivity_label->setToolTip(sensitivity_tool_tip);
+  sensitivity_label->setText(tr("Sensitivity"));
+
+  sensitivity_layout->addWidget(sensitivity_label);
+  sensitivity_layout->addWidget(m_sensitivity_control);
+
+  return sensitivity_layout;
+}
+
+QHBoxLayout* MouseSettingsWidget::CreateSnappingDistanceLayout()
+{
+  QHBoxLayout* snapping_distance_layout = new QHBoxLayout{};
+  m_snapping_distance_control = new QDoubleSpinBox{};
+
+  m_snapping_distance_control->setRange(0.00, 9999.50);
+  m_snapping_distance_control->setDecimals(2);
+  m_snapping_distance_control->setSingleStep(0.5);
+  m_snapping_distance_control->setWrapping(true);
+  QString snapping_distance_tool_tip{
+      tr("Distance (in pixels, can do partial pixels) around notches where cursor snaps to notch."
+         "\nMouse is only locked into notches and snapping will "
+         "only occur when the game is started and has focus."
+         "\n\n0.0(default) disables snapping")};
+  m_snapping_distance_control->setToolTip(snapping_distance_tool_tip);
+
+  QLabel* snapping_distance_label = new QLabel{};
+  snapping_distance_label->setText(tr("Snapping Distance"));
+  snapping_distance_label->setToolTip(snapping_distance_tool_tip);
+
+  snapping_distance_layout->addWidget(snapping_distance_label);
+  snapping_distance_layout->addWidget(m_snapping_distance_control);
+  return snapping_distance_layout;
+}
+QHBoxLayout* MouseSettingsWidget::CreateOctagonalMouseJailIsEnabledLayout()
+{
+  QHBoxLayout* octagonal_mouse_jail_is_enabled_layout = new QHBoxLayout{};
+  m_octagonal_mouse_jail_is_enabled_control = new QCheckBox{};
+  QString octagonal_mouse_jail_is_enabled_tool_tip = {tr(
+      "Locks the mouse cursor into a simulated octagonal gate. \n(Think of the edge around a "
+      "gamecube controller stick)\nLow sensitivity(under 2.0ish) may not feel like anything is "
+      "happening. \nSomething is happening, the mouse gate is just REALLY big. \n(Try moving your "
+      "mouse to the edge of the screen)"
+      "\n\nSnapping is disabled if this is disabled"
+      "\n\nLocking and snapping only occur when a game is started and focused"
+      "\n\nYou can make the mouse cursor invisible at:"
+      "\nConfig->Interface Tab->Mouse Cursor Visibility->Never"
+      "\n(Lock Mouse Cursor locks mouse into the game window. \nDifferent and unreleated to the "
+      "mouse notches)")};
```

Comment:
I think this is a better way to present the information
```
Locks the mouse cursor into a simulated octagonal gate; like the one around
the joystick of a Gamecube controller (does not affect the functionality of the
mouse if the game window doesn't have focus).
This setting won't have a noticeable effect if the mouse sensitivity is low. 
Disabling this setting will also disable "Notch Snapping".
The setting "Lock Mouse Cursor" works independently of this, since it's
meant to apply to the entire game window.
```
The bit about the cursor visibility is irrelevant.

In any case, this setting should (logically) come before the snapping distance. That way we can remove the line "Disabling this setting will also disable "Notch Snapping"."

Reply:
Almost forgot, you got to make sure that when the setting is disabled, it greys out the "snapping distance" part.